addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -207,7 +207,14 @@ function factory(dependencies) {
             }
             thread.open();
             if (this.messaging.device.isMobile && thread.channel_type) {
-                this.update({ activeMobileNavbarTabId: thread.channel_type });
+                const channelTypeTab = {
+                    chat: 'chat',
+                    group: 'chat',
+                    channel: 'channel',
+                    mailbox: 'mailbox',
+                    all: 'all',
```

Comment:
There is no such thing as `mailbox` and `all` `channel_type`, those keys are unnecessary.

Reply:
Hum indeed, I was a bit too fast to approve. `tabId` valid values are `mailbox`, `channel`, `chat`... and `livechat`.

Before PR code that uses `thread.channel_type` actually cover all but `mailbox`. A bit strange the code does pass here with `mailbox` and it "works" thanks to default value for active tab id...

With these changes, it should break with `livechat`, and `all`/`mailbox` are basically deadcode because no valid `channel_type`.